Detectors/ITSMFT/ITS/workflow/src/TrackerSpec.cxx

```diff
@@ -135,7 +112,7 @@ void TrackerDPL::run(ProcessingContext& pc)
       roFrame++;
     }
   } else {
-    o2::ITS::IOUtils::loadEventData(event, &clusters, labels.get());
+    ITS::IOUtils::loadEventData(event, &clusters, labelsPtr);
```

Comment:
@mpuccio Do not know how to comment on lines that were not changed...
But the lines for "TRACKSMCTR" and "ITSTrackMC2ROF" outputs below, should not they now be guarded by "if (mIsMC)" as well ?

Reply:
@iouribelikov , @mpuccio:  Just saw this, yes, they should be guarded. But one cannot guard ``auto...`` by ``if () {}``, so one should define outside a pair of raw and unigue pointers and get from pointer from the pc.input, rather than the reference, see https://github.com/AliceO2Group/AliceO2/blob/afcabb81ff3df49f90c799f5765654eb2ebcaf47/Detectors/GlobalTrackingWorkflow/src/TPCITSMatchingSpec.cxx#L220 .